sqliterepo/election.c

```diff
@@ -1352,7 +1352,11 @@ member_json (struct election_member_s *m, GString *gs)
 	g_string_append_c (gs, ',');
 	OIO_JSON_append_str (gs, "type", m->inline_name.type);
 	g_string_append_c (gs, ',');
-	OIO_JSON_append_str (gs, "zk", m->key);
+	gchar *full_key = sqlx_sync_zk_full_key_path(m->sync, m->key);
```

Comment:
:+1: 

Maybe we should also show which zk cluster is used for this election ? (however it seems to overlaps `zk_server`)